packages/backend/README.md

```diff
@@ -125,7 +125,10 @@ ETHEREUM_ALCHEMY_API_KEY=
 # ARBITRUM_TVL_ENABLED=
 # ARBISCAN_API_KEY=
 # ARBITRUM_PROVIDER_URL=
-
+## Enable/disable detailed-tvl endpoint exposure - defaults to true
+# DETAILED_TVL_ENABLED=
```

Comment:
We should not default to true. Best practice is to always default to false and have the user provide non-falsy values.
Also for booleans in this file we could provide `=true` instead of just `=` since that would be the only possible version.

Reply:
Fair catch https://github.com/l2beat/l2beat/pull/1776/commits/36dc613c2e0e1fbd2fa98cf8d781b84e5a79ca49